targets/XPlatCppSdk/source/cppsdk/include/playfab/PlayFabCallRequestContainerBase.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <unordered_map>
+
+namespace PlayFab
+{
+    class CallRequestContainerBase;
+    typedef std::shared_ptr<void> SharedVoidPointer;
```

Comment:
trying to read a shared_ptr is easier than trying to understand a typedef - that's an extra layer of dereferencing in my mind - but up to you.